Scripts/script-CommonServer.yml

```diff
@@ -1410,7 +1410,8 @@ script: |-
       url: 'URL(val.Data && val.Data === obj.Data)',
       domain: 'Domain(val.Name && val.Name === obj.Name)',
       cve: 'CVE(val.ID && val.ID === obj.ID)',
-      email: 'Account.Email(val.Address && val.Address === obj.Address)'
+      email: 'Account.Email(val.Address && val.Address === obj.Address)',
+      dbotscore: 'DBotScore(val.Indicator === obj.Indicator && val.Vendor === obj.Vendor)',
```

Comment:
Please verify it exists first by `val.Indicator && val.Indicator ===...`

Reply:
DBotScore path should be:
`DBotScore` and that is it

Reply:
It's already exists in CommonServerPython, I just added it to CommonServer. shouldn't we use it?